style.css

```diff
@@ -259,3 +265,104 @@ h2 {
 .underline {
   list-style: none;
 }
+
+.contactMe {
+  background-color: #6070ff;
+  border-radius: 5rem 0 0 0;
+  background-image: url(./img/background/contactFormBack.svg);
+  background-repeat: no-repeat;
+  background-position: right 0;
+  background-size: 50vw;
+  background-position-x: 12rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  size: 100vw;
+  color: #172b4d;
+  font-weight: 700;
+  font-size: 2.5rem;
+  margin-top: 10rem;
+}
```

Comment:
- [ ] Please 🙏🏻 adhere to the project requirement or `Figma` design, according to which your design layout is a little off, there is something in your design which makes your design overflow and makes your design looks unprofessional. Your contact section's items are not aligned properly. Kindly take a look at the below design comparison and the requirements and I also attached a video for pointing out the issue.

### Requirement
![image](https://user-images.githubusercontent.com/66476329/236188683-4224cc77-237d-413d-9942-91297c87687b.png)

### Design Comparison
<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![image](https://github.com/DamianDib9/personal-portfolio/assets/66476329/486e4740-cf92-41cf-b28e-0e157db9e10f)

</td>
<td>

![Screenshot from 2023-07-16 00-26-33](https://github.com/DamianDib9/personal-portfolio/assets/66476329/e4cc134f-12b9-4afc-a1a7-7981ce027fc1)

</td>
</tr>
</table>


### Video
[Screencast from 16-07-23 12:31:13 AM IST.webm](https://github.com/DamianDib9/personal-portfolio/assets/66476329/823105d3-e947-4b26-92ce-9c8f4eb2043a)

Reply:
Great! Fixed that overflow, it was related to the background settings.